cpp/src/arrow/table_test.cc

```diff
@@ -15,20 +15,21 @@
 // specific language governing permissions and limitations
 // under the License.
 
-#include <cstdint>
-#include <memory>
-#include <vector>
+#include "arrow/table.h"
 
 #include <gmock/gmock.h>
 #include <gtest/gtest.h>
 
+#include <cstdint>
+#include <memory>
+#include <vector>
+
```

Comment:
I'm a bit surprised that you needed to reorder these includes, though this is not a problem. Out of curiosity, did an automatic formatter do this?

Reply:
Yes, I think the formatter just reorder it alphabetically.